docker-compose.yml

```diff
@@ -54,8 +56,8 @@ services:
         --plugin aries_cloudagent.messaging.jsonld \
         --inbound-transport http '0.0.0.0' ${AGENT_HTTP_PORT} \
         --outbound-transport http \
-        --endpoint ${AGENT_ENDPOINT} \
-        --profile-endpoint ${BPA_PROFILE_ENDPOINT} \
+        --endpoint http://${BPA_HOST}:${AGENT_HTTP_PORT} \
+        --profile-endpoint https://${BPA_HOST}/profile.jsonld \
```

Comment:
This moves protocol specifics http(s) into the compose file. If someone has different requirements the compose file needs to change.

Reply:
I think this is fine. If you really need this you could add AGENT_ENDPOINT_PROTOCOL and BPA_PROFILE_PROTOCOL env variables